bindings/c/test/mako/stats.hpp

```diff
@@ -89,8 +89,8 @@ class DDSketchMako : public DDSketch<uint64_t> {
 };
 
 class alignas(64) ThreadStatistics {
-	uint64_t conflicts;
-	uint64_t total_errors;
+	uint64_t conflicts{ 0 };
+	uint64_t total_errors{ 0 };
```

Comment:
original PR has an additional field https://github.com/apple/foundationdb/pull/9007/files#diff-531f874c0b88cb4632480e5e2fb24736d5f786d1f392d4283988d24e0a2ced3dR94, skip it here. @jzhou77